src/plugins/pcr_primer_design_for_dna_assembly/src/tasks/PCRPrimerDesignForDNAAssemblyTask.cpp

```diff
@@ -43,6 +45,244 @@
 
 namespace U2 {
 
+//To generate a PCRPrimerDesignForDNAAssemblyTask report.
+namespace ReportUtils {
+using PcrTask = PCRPrimerDesignForDNAAssemblyTask;
+using Reports = PcrTask::UserPrimerReports;
+
+static QString forwardUserPrimerStr();
+static QString reverseUserPrimerStr();
+
+//No validation of columns number when adding rows.
+class UserPrimersTable {
```

Comment:
Please, move it to the separate .h and .cpp files, cuz this one is overfilled already